test/action_controller/serialization_test.rb

```diff
@@ -136,6 +136,13 @@ def render_fragment_changed_object_with_relationship
           render json: like
         end
 
+        def render_object_xml
+          profile = Profile.new(name: 'Name 1', description: 'Description 1', comments: 'Comments 1')
+          with_adapter :flatten_json do
+            render xml: profile
```

Comment:
This would work nicely, name-wise, with https://github.com/rails-api/active_model_serializers/pull/1117/